css-cascade/specificity.css

```diff
@@ -0,0 +1,19 @@
+* {
```

Comment:
all of these selectors should be in reverse order as noted in the instructions. that is to demonstrate that specificity is more powerful than source order.